### ISSUE TITLE
ex 2 test case 4 unable to pass

### ISSUE THREAD

**Issue by Christian1228** [nus-cs2030/2223-s2#147]
![Screenshot (1)](https://user-images.githubusercontent.com/122709629/219392178-0354fca5-bf1f-4753-ae08-af11f672e937.png)
I am unable to pass test case 4 for ex 2 Im not sure why

**Comment by SimonSeeYongsheng** [comment]
https://github.com/nus-cs2030/2223-s2/issues/46 You might want to take a look at this

**Comment by geoffong11** [comment]
There may be an error with your test case 3 method (happened to me as well even though it says I passed test case 3)

**Comment by JustinGohkh** [comment]
the passengers' part thereof also have to be taken into account!

**Comment by meljwang** [comment]
I am facing the exact same error as you, even though I have already accounted for the part thereof for both number of loaders and service time.

**Comment by Christian1228** [comment]
yes i managed to figure it out, my service time for my Big cruise i didnt Math ceil the service time for my numPassengers
